components/logins/src/sync/engine.rs

```diff
@@ -102,7 +103,13 @@ impl LoginsSyncEngine {
                             upstream.guid(),
                             dupe.guid()
                         );
-                        plan.plan_two_way_merge(&dupe, (upstream, upstream_time));
+                        let local_modified = UNIX_EPOCH
+                            + Duration::from_millis(dupe.record.time_password_changed as u64);
```

Comment:
Out of curiosity, why don't we use the `local_modified` column instead here? I get that the struct `EncryptedLogin` doesn't have it as a field (but we could easily pull it from the db if we needed to so I'm curious if there's another reason)

Reply:
I agree we probably should, but I didn't want to change the semantics in this patch, mainly because I was slightly worried the semantics of that column might not be what we want (ie, I didn't want to dig into the semantics of that field and how sync itself updates it, for example)